front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/PipelineController.groovy

```diff
@@ -44,6 +44,18 @@ class PipelineController {
     pipelineDAO.getPipelinesByApplication(application)
   }
 
+  @RequestMapping(value = 'getById/{id:.+}', method = RequestMethod.GET)
```

Comment:
I don't believe we should be explicitly encoding verbs (like `getById`/`getByName`) in query paths ... I would also drop the lookup by name and only support `id`. These are new API endpoints without callers, we don't need to support lookups by the deprecated name.

The API around pipelines is rather inconsistent in that we have `/pipelines/{application}' where it should be `/pipelines/{id}` IMO. The ability to lookup by application would make more sense if `application` where a query parameter on `/pipelines`.

Given that's where we are right now, I would lean towards `/pipelines/{application}/{id}`.